test/integration/main.test.js

```diff
@@ -641,8 +866,7 @@ describe('invite', () => {
 
                 // me is the current context
                 yourChat.invite(ChatEngine.me);
-
-            }, 5000);
+            }, 2000);
```

Comment:
Not sure these are required to pass?

Reply:
I remember you mentioned it helped, but was probably unrelated. I did a lot of timeout config too when I dealt with similar problems.